photosynthesis_metrics/vif.py

```diff
@@ -30,17 +30,18 @@ def _gaussian_kernel2d(kernel_size: int = 5, sigma: float = 2.0) -> torch.Tensor
     return kernel
 
 
-def vif_p(prediction: torch.Tensor, target: torch.Tensor,
-          sigma_n_sq: float = 2.0, data_range: Union[int, float] = 1.0) -> torch.Tensor:
+def vif_p(prediction: torch.Tensor, target: torch.Tensor, sigma_n_sq: float = 2.0,
```

Comment:
Please, add input validation and adjustments to `vif_p`. As a result, there will be no need in validation check in `VIFLoss().forward()`, while `VIFLoss().compute_metric()`, which consists of one string, can be moved to `forward()`.